cloud/vsphere/machineactuator.go

```diff
@@ -191,13 +192,13 @@ func (vc *VsphereClient) cleanUpStagingDir(machine *clusterv1.Machine) error {
 }
 
 // Builds and saves the startup script for the passed machine and cluster.
```

Comment:
Update the comment to include what is returned.

Reply:
updated.

Reply:
updated.